charts/yawol-controller/Chart.yaml

```diff
@@ -3,5 +3,5 @@ description: Helm chart for yawol-controller
 name: yawol-controller
 sources:
   - https://github.com/stackitcloud/yawol
-version: "0.25.0"
-appVersion: v0.25.0
+version: "0.26.0"
+appVersion: v0.26.0-0
```

Comment:
Why "0-0"?

Reply:
Because it's not a final release.

But I changed it wrong. It's now fixed.